tests/JWT.Tests/DecodeTests.cs

```diff
@@ -1,14 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
+
+#if !NETCOREAPP1_0
 using System.Web.Script.Serialization;
+#endif
 
 using FluentAssertions;
 
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit;
```

Comment:
yes.  xunit is like the default dotnetcore unit testing framework.   All the examples Ive seen use it and I wouldn't have a clue how to setup anything else :)